data/assets/scene/solarsystem/missions/euclid/mission.asset

```diff
@@ -0,0 +1,85 @@
+  Milestones = {
+    {
+      Name = "Launch",
+      Date = "2023 JUL 01 15:12:00"
+    },
+    {
+      Name = "Quality Release Q1",
+      Date = "2024 DEC 01"
+    },
+    {
+      Name = "Data Release DR1",
+      Date = "2025 DEC 01"
+    },
+    {
+      Name = "Quality Release Q2",
+      Date = "2026 DEC 01"
+    },
+    {
+      Name = "Data Release DR2",
+      Date = "2027 DEC 01"
+    },
+    {
+      Name = "Quality Release Q3",
+      Date = "2028 DEC 01"
+    },
+    {
+      Name = "Quality Release Q4",
+      Date = "2029 DEC 01"
+    },
+    {
+      Name = "Data Release DR3",
+      Date = "2030 DEC 01"
+    }
+  },
+  Phases = {
+    {
+      Name = "Cruise phase",
+      TimeRange = { Start = "2023 JUL 01 15:12:00", End = "2023 AUG 01" },
+      Description = "Following a cruise of 30 days, Euclid will operate in a large halo orbit around the L2 point of the Sun-Earth system, which is located 1.5 million km from the Earth in the anti-Sun direction. The orbit will not suffer from any Earth eclipses and will have a radius of 1 million kilometers. An operational lifetime of 6 years is planned."
+    },
+    {
+      Name = "Commissioning & Performance Verification phase",
+      TimeRange = { Start = "2023 AUG 01", End = "2023 OCT 01" },
+      Description = ""
+    },
+    {
+      Name = "Operational phase",
+      TimeRange = { Start = "2023 AUG 01", End = "2028 AUG 01" },
+      Description = [[Euclid was selected for implementation in June 2012 by ESA's Science Programme Committee. It's main scientific objectives are:
+
+        - To investigate whether Dark Energy is real, or whether the apparent acceleration of the Universe is caused by a breakdown of General Relativity on the largest scales, or a failure of our assumption that the universe is more or less uniform in density.
+        - If dark energy is real, to investigate whether it is a constant energy spread across space, which would make it the Cosmological Constant found in Einstein's General Relativity, or a new force of nature that evolves with the expansion of the Universe.
+        - To investigate the nature of Dark Matter, the mass of neutrino particles and whether there are other, so-far undetected fast moving particle species in the Universe.
+        - To investigate the conditions in the Universe after the Big Bang that seeded the large-scale structure of the universe that we observe today.]]
+    }
+  }
+}
+
+
+asset.onInitialize(function()
+  openspace.loadMission(Mission)
+end)
+
+asset.onDeinitialize(function()
+  openspace.unloadMission(Mission.Name)
+end)
+
+
+
+asset.meta = {
+  Name = "Euclid Mission",
+  Version = "1.0",
+  Description = "This asset contains the mission profile information for ESA's Euclid mission.",
```

Comment:
Maybe include the sources of the information in the mission in this description?  I always end up wondering where the information comes from